test/Graphics/CircleShape.cpp

```diff
@@ -0,0 +1,94 @@
+        CHECK(circle.getPoint(17) == ApproxVec(8.898950577, 28.703182220));
+        CHECK(circle.getPoint(18) == ApproxVec(6.183218956, 27.135253906));
+        CHECK(circle.getPoint(19) == ApproxVec(3.852828979, 25.036960602));
+        CHECK(circle.getPoint(20) == ApproxVec(2.009618759, 22.500000000));
+        CHECK(circle.getPoint(21) == ApproxVec(0.734152794, 19.635255814));
+        CHECK(circle.getPoint(22) == ApproxVec(0.082171440, 16.567928314));
+        CHECK(circle.getPoint(23) == ApproxVec(0.082171440, 13.432073593));
+        CHECK(circle.getPoint(24) == ApproxVec(0.734151840, 10.364746094));
+        CHECK(circle.getPoint(25) == ApproxVec(2.009618759, 7.500000000));
+        CHECK(circle.getPoint(26) == ApproxVec(3.852827072, 4.963040352));
+        CHECK(circle.getPoint(27) == ApproxVec(6.183218002, 2.864748001));
+        CHECK(circle.getPoint(28) == ApproxVec(8.898950577, 1.296818733));
+        CHECK(circle.getPoint(29) == ApproxVec(11.881320953, 0.327786446));
+    }
+
+    SUBCASE("Radius and point count constructor")
+    {
+        const sf::CircleShape circle(5.f, 8);
+        CHECK(circle.getRadius() == 5.f);
+        CHECK(circle.getPointCount() == 8);
+        CHECK(circle.getPoint(0) == ApproxVec(5.000000000, 0.000000000));
+        CHECK(circle.getPoint(1) == ApproxVec(8.535533905, 1.464465857));
+        CHECK(circle.getPoint(2) == ApproxVec(10.000000000, 5.000000000));
+        CHECK(circle.getPoint(3) == ApproxVec(8.535533905, 8.535533905));
+        CHECK(circle.getPoint(4) == ApproxVec(5.000000000, 10.000000000));
+        CHECK(circle.getPoint(5) == ApproxVec(1.464466095, 8.535533905));
+        CHECK(circle.getPoint(6) == ApproxVec(0.000000000, 4.999999523));
+        CHECK(circle.getPoint(7) == ApproxVec(1.464465857, 1.464466572));
+    }
+
+    SUBCASE("Set point count")
+    {
+        sf::CircleShape circle(4.f, 10);
+        circle.setPointCount(4);
+        CHECK(circle.getRadius() == 4.f);
+        CHECK(circle.getPointCount() == 4);
+        CHECK(circle.getPoint(0) == ApproxVec(3.999999762, 0.000000000));
+        CHECK(circle.getPoint(1) == ApproxVec(8.000000000, 4.000000000));
+        CHECK(circle.getPoint(2) == ApproxVec(3.999999762, 8.000000000));
+        CHECK(circle.getPoint(3) == ApproxVec(0.000000000, 3.999999762));
+    }
+
+    SUBCASE("Set radius")
+    {
+        sf::CircleShape circle(1.f, 6);
+        circle.setRadius(10.f);
+        CHECK(circle.getRadius() == 10.f);
+        CHECK(circle.getPointCount() == 6);
+        CHECK(circle.getPoint(0) == ApproxVec(10.000000000, 0.000000000));
+        CHECK(circle.getPoint(1) == ApproxVec(18.660253525, 5.000000000));
+        CHECK(circle.getPoint(2) == ApproxVec(18.660253525, 15.000000000));
+        CHECK(circle.getPoint(3) == ApproxVec(10.000000000, 20.000000000));
+        CHECK(circle.getPoint(4) == ApproxVec(1.339746475, 15.000000000));
+        CHECK(circle.getPoint(5) == ApproxVec(1.339745522, 5.000000000));
+    }
+}
```

Comment:
Since it's rather [prominently featured in the tutorial](https://www.sfml-dev.org/tutorials/2.5/graphics-shape.php#built-in-shape-types), it might make sense to test some of the regular polygons (triangle, square, hexagon).

Reply:
4, 6, and 8-sided polyhedra are already being tested so I added a new test case for an equilateral triangle.